docs/src/components/ApidocLink/ApidocLink.js

```diff
@@ -105,13 +105,11 @@ export const createFromType = (type, defProps) => {
 const ApidocLink = ({ logo, renderLink }) => (
   <aside className={getClassName('bpkdocs-apidoc-link')}>
     {logo && (
-      /* eslint-disable backpack/use-components */
       <img
         className={getClassName('bpkdocs-apidoc-link__logo')}
         src={`/${logo.src}`}
         alt={logo.alt}
       />
-      /* eslint-enable backpack/use-components */
     )}
```

Comment:
Shouldn't we just make this a BPKImage? We know the dimensions so should be possible?

Reply:
😉 

<img width="572" alt="Screen Shot 2020-12-14 at 14 22 30" src="https://user-images.githubusercontent.com/73652/102092283-ced27100-3e17-11eb-8384-05300da6fccb.png">

Reply:
🤦

Reply:
I've decided to not update any components that we expect to remove soon in favour of MDX, so technically we should but I'm going to leave it.